docs/networks.md

```diff
@@ -40,3 +40,26 @@ Here are the illegal ones:
 
 - (project, all, yes): In the same vein, user-created public networks are also
       not allowed.
+
+
+# Channels
+
+In some contexts it's possible to connect more than one logical network
+to a physical interface, and somehow "tag" the traffic, such that the
+host can manage these separate networks. One example of this is 802.1q
+VLANs.
+
+The common abstraction for this is called a "channel." A channel is a
+driver-specific identifier which describes how traffic will be tagged on
+the network. It is used by HaaS in the following ways:
+
+- The `node_connect_network` api call accepts an optional channel argument,
+  which determines how the network will be associated with the given
+  port. The headnode equivalents may support this eventually, but do not
+  as of yet.
+
+  If this argument is left out, a sensible default will be chosen.
```

Comment:
Quoting George Silvis, III (2015-04-02 13:27:31)
 
>    I would say that the default should be to put the network traffic on
>    it, unmodified. Being sure that you can simply listen to the ethernet
>    cable for that traffic, whatever the driver is, seems very useful to
>    me.

Yeah, I think "native" or equivalent is almost always going to be the 
right default. I want to keep this abstract though.